src/random.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+function random() {
+  let checkedValue = false;
+  const randomNum = Math.ceil(Math.random() * 10000);
+
+  for (let i = 0; i < randomNum.length; i++) {
+    for (let q = 0; q < randomNum.length; q++) {
+      if (randomNum[i] === randomNum[q]) {

```

Comment:
The if condition will always be true when i equals to q. You need to exclude the case when i equals to q.